InnerEye/ML/lightning_base.py

```diff
@@ -161,11 +161,15 @@ def setup(self) -> None:
                 unique_ids = set(split_data[CSV_SUBJECT_HEADER])
                 for patient_id in unique_ids:
                     rows = split_data.loc[split_data[CSV_SUBJECT_HEADER] == patient_id]
+                    allow_incomplete_labels = False
+                    if isinstance(self.config, SegmentationModelBase):
+                        allow_incomplete_labels = self.config.allow_incomplete_labels  # type: ignore
```

Comment:
I think this is already implied by the check in line 148? If mypy complains, then replace the check in 148 with the isinstance... check

Reply:
Good point, I'd missed the similar check on 148. Fixed.